### PR TITLE
ocs-operator: migrate to red-hat-storage

### PR DESCRIPTION
**Pull request by jarrpa**

### CODE REVIEW

File: ci-operator/config/openshift/ocs-operator/OWNERS [hunk 1]
```diff
@@ -1,18 +0,0 @@
-# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
```
Comment (BlaineEXE) [inline ci-operator/config/openshift/ocs-operator/OWNERS:1]: Just double checking that there are no other files present in the dir if the `OWNERS` file is being removed.

**Review by BlaineEXE — APPROVED** [review]
Looks good. Just double checking that since we are removing ` ci-operator/config/openshift/ocs-operator/OWNERS` there are no other files present in that dir. We don't want to have to modify something here in the near future and find we aren't able.

File: ci-operator/config/openshift/ocs-operator/OWNERS (hunk 1 above)
Comment (jarrpa) [reply ci-operator/config/openshift/ocs-operator/OWNERS:1]: Yup, dir's gone.

**Pull request merged**